common/src/main/java/org/opensearch/ml/common/breaker/MLCircuitBreakerService.java

```diff
@@ -62,6 +62,8 @@ public MLCircuitBreakerService init() {
         // Register memory circuit breaker
         registerBreaker(BreakerName.MEMORY, new MemoryCircuitBreaker(this.jvmService));
         log.info("Registered ML memory breaker.");
+        registerBreaker(BreakerName.DISK, new DiskCircuitBreaker());
```

Comment:
Suggest set the `diskDir` of `DiskCircuitBreaker` as OpenSearch data folder path. Check https://github.com/ylwu-amzn/ml-commons/blob/custom_model/common/src/main/java/org/opensearch/ml/common/breaker/MLCircuitBreakerService.java#L67, and https://github.com/ylwu-amzn/ml-commons/blob/custom_model/plugin/src/main/java/org/opensearch/ml/plugin/MachineLearningPlugin.java#L225

Reply:
cool. Just updated it with the OS data folder.